src/kOS/Suffixed/OrbitInfo.cs

```diff
@@ -7,7 +7,7 @@
 
 namespace kOS.Suffixed
 {
-    public class OrbitInfo : Structure, IDumperWithSharedObjects
+    public class OrbitInfo : SerializableStructure, IHasSharedObjects
```

Comment:
I see a few fields that look like they wouldn't survive a serialization and reloading, like the `nextPatch` or `transition` information.